### DISCUSSION TITLE
OpenTelemetry .NET Auto-Instrumentation not showing database queries

### DISCUSSION THREAD

**Discussion by NamitaJaokar** [open-telemetry/community#939] [Community Discussions]
Hi,

I have configured **OpenTemeletry .Net Auto instrumentation** which is a pre-release version  with a sample mvc application that uses postgresql databse with EntityFrameworkCore instrumentation. **The agent works with zipkin exporter** and I am able to view the traces in the zipkin dashboard.
However, I want the database statements to be seen which currently cannot be found in the traces. Which environment variable do I need to set so as to get the database statements in the zipkin dashboard. In current configurations, I can only view db.name="dbname" , db.type="postgres", name="postgres.query" as database attributes.
The same application when manually instrumented with OpenTelemetry's **AddEntityFrameworkCoreInstrumentation(options.setDbStatementForText=true)** was able to give the desired result.

What configurations will be required to get the db.statement that will have the select/update/insert/ queries as the value.

**Comment by trask** [comment]
hi @NamitaJaokar! I'd recommend posting to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation

**Reply by NamitaJaokar** [reply]
Thank you @trask , I have posted my query in the issues section of OTEL-dotnet-Instrumentation.
Reference:
 https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/305
